.cirrus.yml

```diff
@@ -50,6 +50,28 @@ task:
     - bash ci/script.sh
   before_cache_script: rm -rf $CARGO_HOME/registry/index
 
+
+# macOS aarch64 requires a more recent toolchain than the other macOS targets.
+task:
+  name: macOS aarch64
+  env:
+    TARGET: aarch64-apple-darwin
+    TOOLCHAIN: stable
+    DISABLE_TESTS: 1
+  osx_instance:
+    image: big-sur-xcode
+  setup_script:
+    - curl --proto '=https' --tlsv1.2 -sSf -o rustup.sh https://sh.rustup.rs
+    - sh rustup.sh -y --profile=minimal --default-toolchain $TOOLCHAIN
+    - . $HOME/.cargo/env
+    - rustup target add $TARGET
+    - bash ci/install.sh
+  script:
+    - . $HOME/.cargo/env
+    - bash ci/script.sh
+  before_cache_script: rm -rf $CARGO_HOME/registry/index
+  allow_failures: true
```

Comment:
Please remove this line.  You read the README correctly, but the README has grown out of sync with actual practice.  I'll fix the README.